src/main/resources/org/jenkinsci/plugins/credentialsbinding/impl/BindingStep/help.html

```diff
@@ -73,13 +73,8 @@
 foo'bar
 </code></pre>
 <p>
-    Mangled secrets are not detected by Jenkins and will appear in plain text in the log file, in this case as:
-</p>
-<pre><code>+ echo 'foo'\''bar'
-****</code></pre>
-<p>
-    This particular issue can be worked around by turning off echo with <code>set +x</code>,
-    or simply avoiding possible shell metacharacters in secrets.
+    Mangled secrets can only be detected on a best effort basis using the <code>PatternMaskerProvider</code>
```

Comment:
Not something I would include in user documentation: Nobody cares how it's done.